src/Api/Response/JsonWebKeySetCollection.php

```diff
@@ -19,16 +19,16 @@ public function __construct(array $authKeys)
 
     public function getByCryptographicAlgorithm(string $algorithm): ?JsonWebKeySet
     {
-        if (!CryptographicAlgorithmEnum::isSupported($algorithm)) {
+
```

Comment:
The only change is changed formatting.

```suggestion
    public function getByCryptographicAlgorithm(string $algorithm): ?JsonWebKeySet
    {
        $result = $this->authKeys[$algorithm] ?? null;
        if (!$result) {
            throw new UnsupportedCryptographicAlgorithmException(
                sprintf(
                    'Cryptographic algorithm `%s` is not supported.',
                    $algorithm
                )
            );
        }

        return $result;
    }
```